docs/develop/understanding-the-javascript-api-for-office.md

```diff
@@ -28,31 +28,89 @@ For more details around the Office.js CDN, including how versioning and backward
 
 **Applies to:** All add-in types
 
-Office.js provides an initialization event which gets fired when the API is fully loaded and ready to begin interacting with the user. You can use the **initialize** event handler to implement common add-in initialization scenarios, such as prompting the user to select some cells in Excel, and then inserting a chart initialized with those selected values. You can also use the initialize event handler to initialize other custom logic for your add-in, such as establishing bindings, prompting for default add-in settings values, and so on.
+Your code must not call any Office.js APIs until the library is fully loaded. There are two events that your code can handle to ensure that the library is loaded. They are described in the sections below. We recommend that you use the newer, more flexible, `Office.onReady`. But the older `Office.initialize` is still supported. The main differences between them:
 
-At a minimum, the initialize event would look like the follow example:     
+- You can assign only one handler to `Office.initialize`. But you can assign multiple handlers, in different places in your code, to `Office.onReady`.
```

Comment:
Not so much "assign multiple handlers" but "you can use `Office.onReady` anytime you want, calling it as a method rather than setting a specific variable to your initialization function`